### PR TITLE
Add release notes for 0.5.14

### PR DESCRIPTION
**Pull request by Caleb-Hill**
Signed-off-by: Caleb Hill <hill@bitwise.io>

### CODE REVIEW

**Review by peterschwarz — DISMISSED** [review]

**Review by peterschwarz — DISMISSED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+
+* Add any_pending_migrations to postgres and sqlite migration modules
+
+* Fix identity comparison in PeerManager to decide between inbound/outboud
+  connections. This comparison must now be between the connection's identity
+  and the required local authorization used when requesting the connection
+  originally.
+
+* Stabilize `admin-service-count`. This makes `count_circuits` and
+  `count_proposals`available on the     `AdminServiceStore`.
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:27]: Looks like spurious white-space after "the"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+* User configurable logging with stdout, stderr, file, and rotating file log
+  targets as well as fine grain control over where each log statement goes.
+
+### libsplinter
+
+* Replace the use of a Vec<u8> to represent a public key in the
+  PeerAuthorizationToken, ConnectionAuthorizationType, Identity, and in the
+  AdminService.
+
+* Add any_pending_migrations to postgres and sqlite migration modules
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:19]: Add code back-ticks to the `any_pending_migrations`

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+  originally.
+
+* Stabilize `admin-service-count`. This makes `count_circuits` and
+  `count_proposals`available on the     `AdminServiceStore`.
+
+### splinterd
+
+* Detect pending migrations and abort if migrations are required
+
+* Add `--peering-key` option to the splinterd to set which key will be used
```
Comment (peterschwarz) [inline RELEASE_NOTES.md:33]: The peering ones are under an experimental feature, which should be noted in both entries

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Lot of weird spaces were added, I removed all but one. Weird

File: RELEASE_NOTES.md (hunk 2 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: fixed

File: RELEASE_NOTES.md (hunk 3 above)
Comment (Caleb-Hill) [reply RELEASE_NOTES.md]: Added some language to that effect on both entries

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
 # Release Notes
 
+## Changes in Splinter 0.5.14
+
+### Highlights
+
+* Splinterd will not start if there are database migrations that need to be run
+  and will give information about how to upgrade the database.
+
+* User configurable logging with stdout, stderr, file, and rotating file log
```
Comment (agunde406) [inline RELEASE_NOTES.md:10]: This should also be added to the splinterd section (with different wording)

**Review by agunde406 — CHANGES_REQUESTED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by isabeltomb — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**